app/src/main/java/ir/ac/ut/berim/TestScrollActivity.java

```diff
@@ -57,12 +55,11 @@ protected void onCreate(Bundle savedInstanceState) {
 
     @Override
     public void onScrollChanged(int scrollY, boolean firstScroll, boolean dragging) {
-//        ViewHelper.setTranslationY(mBackgroundImage, -scrollY / 2);
-//        ViewHelper.setTranslationY(listView, -scrollY / 2);
-//        mtextView.setText(temp+","+scrollY  );
-//        findViewById(R.id.placeDescription).getLayoutParams()
-//        LayoutParams lp = (LayoutParams) listView.getLayoutParams();
-//        lp.height += scrollY/2;
+        if(listView.getChildAt(1).getTop()<85)
```

Comment:
adad ha ro az tu resource ha begir, ruye gushi haye mokhtalef bayad fargh kone in adada, alan 85 ru resolution gushieh u kar mide faghat.